crates/cxx-qt-build/src/cpp_file.rs

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
maybe use this branch to update the CHANGELOG too for any build system related fun